docs/migration-guides/qiskit-runtime.mdx

```diff
@@ -53,14 +71,25 @@ See these topics for instructions:
 -  [Update code that calculates expectation values](qiskit-runtime-examples#estimator-algorithm)
 -  [Common use cases (basic, parameterized, and dynamic circuits)](qiskit-runtime-use-case)
 
-
-### Step 3a: replace any `backend.run` options with `qiskit_ibm_runtime` options.
+### Step 3a: Replace any `backend.run` options with `qiskit_ibm_runtime` options
 
 See the following topics for instructions:
 
 - [Migrate options](qiskit-runtime-options)
 - Common use cases [Options section](qiskit-runtime-use-case#migrate-options)
 
+### Step 3b: If you want to access a 3rd-party provider that only exposes `backend.run`, you can still follow a primitive-compatible your workflow using the Qiskit backend primitives
+
+See the following topic for instructions:
+
+- [Migrate using backend primitives](qiskit-backend-primitives)
+
+## Information to share with your users
+
+The following migration guides can help users with the following tasks:
+
+- [Transition to the Qiskit Runtime provider](/migration-guides/qiskit-runtime)
+- [Update user code to the V2 primitives interface](/migration-guides/v2-primitives)
```

Comment:
I'm not sure if this is needed... given it's a migration guide for backend.run to primitives, readers should not have used v1 primitives.